test/TestAssets/XUnitAdapterPerfTestProject/XUnitAdapterPerfTestProject.csproj

```diff
@@ -2,7 +2,7 @@
   <!-- Package dependency versions -->
   <Import Project="..\..\..\scripts\build\TestPlatform.Dependencies.props" />
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
```

Comment:
Latest Xunit adapter supports `net451`?

Reply:
Good catch Sathya, No they don't. I have changed it. Thanks.